spring-integration-mongodb/src/test/java/org/springframework/integration/mongodb/outbound/MongoDbStoringMessageHandlerTests.java

```diff
@@ -49,29 +50,38 @@
  */
 public class MongoDbStoringMessageHandlerTests extends MongoDbAvailableTests {
```

Comment:
Yeah... I know we need to come up with an extension for JUnit 5, or just try to rely on the embedded MongoDB.
Different story, though...
Meanwhile let's stick with JUnit 4 here for a while!

Reply:
I'll have a PR in a minute